install/kubernetes/Makefile

```diff
@@ -0,0 +1,32 @@
+# Variables
+SCRIPT_DIR := $(shell dirname "$(realpath $(lastword $(MAKEFILE_LIST)))")
+TETRAGON_CHART := $(SCRIPT_DIR)/tetragon
+
+# Aliases
+define HELM
+	docker run --rm -v $(TETRAGON_CHART):/apps alpine/helm:3.3.4
+endef
+
+define KUBECONFORM
+	docker run --rm -i ghcr.io/yannh/kubeconform:v0.6.4-alpine@sha256:e68a0b638c6e9b76f1b7d58b4ec94340ef3b6601db25b2e40b29e3ac2d68e4bf
+endef
+
+define HELM_DOCS
+	docker run --rm -v $(TETRAGON_CHART):/helm-docs -u $$(id -u) jnorwood/helm-docs:v1.11.0@sha256:66c8f4164dec860fa5c1528239c4aa826a12485305b7b224594b1a73f7e6879a
+endef
+
+# Targets
+.PHONY: all lint validate docs
+
+all: lint validate docs
+
+lint:
+	$(HELM) dependency update .
```

Comment:
I would rather have a separate target (e.g. `deps`) for this as it's not exactly a part of linting.